### DISCUSSION TITLE
Set playernpc vissible

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#2055] [Help] [answered]
Hey, when I create a player npc ( https://paste.learnspigot.com/umoxugoyos.kotlin )
I cannot turn the invisible off in the init.
I have to do this in another function that will run after the class has been created.

Thats why I have to do this: https://paste.learnspigot.com/mufoquyova.kotlin

Is it possible to set the visibility always to visible if the npc is in the same instance?

**Comment by cosrnic** [comment, marked as answer]
Try removing `isInvisible = true` from the `updateNewViewer` method or making it `!(player.getInstance().equals(this.getInstance))` maybe

**Reply by DevBlokJochem** [reply]
Welp, it was that easy
